ash/src/vk.rs

```diff
@@ -49,4 +49,16 @@ pub trait Handle {
     const TYPE: ObjectType;
     fn as_raw(self) -> u64;
     fn from_raw(_: u64) -> Self;
+
+    /// Returns whether the handle is a `NULL` value.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use ash::vk::{Handle, Instance};
+    ///
+    /// let instance = Instance::null();
+    /// assert!(instance.is_null());
+    /// ```
+    fn is_null(&self) -> bool;
```

Comment:
Like the other methods, this should take `self` by value (see e.g. [`pointer::is_null`](https://doc.rust-lang.org/stable/std/primitive.pointer.html#method.is_null).

Reply:
Good catch, I did not even notice that!